lib/obeya/client.rb

```diff
@@ -66,8 +86,54 @@ def ticket_types
       end
     end
 
+    def tickets_in_named_bin(bin_name)
+      named_bin = bin(bin_name)
+      raise "Bin #{bin_name} not found" unless named_bin
+      tickets_in_bin(named_bin.id)
+    end
+
+    def tickets_in_bin(bin_id)
+      ticket_type_map = Hash[ticket_types.map { |t| [t.id, t] }]
+      bin_map = Hash[bins.map { |b| [b.id, b] }]
+
+      @bin_tickets[bin_id] ||= begin
+        get("/tickets?bin_id=#{bin_id}").map do |ticket_data|
+          Ticket.from_obeya(ticket_data, ticket_type_map, bin_map, custom_fields)
+        end
+      end
+    end
+
+    def matching_tickets_in_named_bin(bin_name, title_matcher)
+      named_bin = bin(bin_name)
+      raise "Bin #{bin_name} not found" unless named_bin
+      matching_tickets_in_bin(named_bin.id, title_matcher)
+    end
+
+    def matching_tickets_in_bin(bin_id, title_matcher)
+      tickets = tickets_in_bin(bin_id)
+      case title_matcher
+      when String
+        tickets.select { |t| t.title == title_matcher }
+      when Regexp
+        tickets.select { |t| t.title =~ title_matcher }
+      end
+    end
+
+    # Get custom fields as an array of id => {id, name, type}
+    def custom_fields
```

Comment:
Yeah, maybe you are right. I tend to have the smallest possible interface. But this is just a wrapper about what is provided by the API. So 👍